client/lib/credit-card-details/validation.js

```diff
@@ -72,14 +73,27 @@ validators.validCreditCardNumber = creditCardValidator( 'validCardNumber' );
 validators.validCvvNumber = creditCardValidator( 'validCvc' );
 
 validators.validExpirationDate = creditCardValidator(
+	'notExpired',
 	'validExpirationMonth',
-	'validExpirationYear'
+	'validExpirationYear',
 );
 
+validators.required = {
```

Comment:
This seems to be copied by mistake here? It looks like it is now twice in the file